pinecone/manage.py

```diff
@@ -194,14 +190,14 @@ def describe_index(name: str):
     """
     api_instance = _get_api_instance()
     response = api_instance.describe_index(name)
-    db = response['database']
-    ready = response['status']['ready']
-    state = response['status']['state']
-    return IndexDescription(name=db['name'], metric=db['metric'],
-                            replicas=db['replicas'], dimension=db['dimension'], shards=db['shards'],
-                            pods=db.get('pods', db['shards'] * db['replicas']), pod_type=db.get('pod_type', 'p1'),
-                            status={'ready': ready, 'state': state}, metadata_config=db.get('metadata_config'),
-                            source_collection=db.get('source_collection', ''))
+    db_status = response.get('status', {})
+    response = response['database'].to_dict()
+    state = db_status.get('state', '')
+    ready = db_status.get('ready', '')
+    status = {'state': state, 'ready': ready}
+    response['status'] = status
+    response_object = IndexDescription(response.keys(), response.values())
```

Comment:
Is it better to just get the dict as a parameter to `IndexDescription` __init__?

Reply:
yeah can just send the dict too here. anything should be okay